Vizzuality/i2i
# Feature/analysis

Description of the PR

Filter and message added to not let the user get analysis unless one of the sector layers or contextual with analysis is selected.

https://projects.invisionapp.com/d/main#/console/18665484/388344482/preview

## Testing instructions

Go to data-portal/geospatial-date/click on a country
http://localhost:3000/data-portal/BGD/fsp-maps?lat=23.56398712845123&layers=%5B%7B%22id%22%3A%22ca3293ef-63f5-462b-ac28-41e82aac21bf%22%2C%22opacity%22%3A1%2C%22visibility%22%3Atrue%7D%2C%7B%22id%22%3A%22d787d894-f7af-47c4-af0f-0849b06686ee%22%2C%22opacity%22%3A1%2C%22visibility%22%3Atrue%7D%2C%7B%22id%22%3A%22478468be-c79f-4472-b910-594f5bb67cf1%22%2C%22opacity%22%3A1%2C%22visibility%22%3Atrue%7D%2C%7B%22id%22%3A%22b1223c4f-7132-42d3-a1cd-90e823adaa5f%22%2C%22opacity%22%3A1%2C%22visibility%22%3Atrue%7D%2C%7B%22id%22%3A%2210004%22%2C%22opacity%22%3A1%2C%22visibility%22%3Atrue%7D%2C%7B%22id%22%3A%22Post%20Offices%22%2C%22opacity%22%3A1%2C%22visibility%22%3Atrue%2C%22visualizationType%22%3A%22normal%22%7D%2C%7B%22id%22%3A%22MFIs%22%2C%22opacity%22%3A1%2C%22visibility%22%3Afalse%2C%22visualizationType%22%3A%22normal%22%7D%2C%7B%22id%22%3A%22Mobile%20Money%20Agent%22%2C%22opacity%22%3A1%2C%22visibility%22%3Atrue%2C%22visualizationType%22%3A%22normal%22%7D%5D&lng=90.26367187500001&zoom=3

Try to get analysis without selecting layers, selecting layers not from sector and selecting sector layers, the message should appear always unless at least one of the sector layers is selected.

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/168661224
